frontend/src/components/taxons/EditTaxon.tsx

```diff
@@ -19,10 +22,14 @@ export const EditTaxonRoute = () => {
     const taxons = useAppSelector(selectTaxonsByID)
     const { id } = useParams();
     const appInitialzed = useAppSelector(selectAppInitialzed)
-    return (id && taxons[id] && appInitialzed) ? <EditTaxon taxon={{ ...taxons[id] }} /> : null
+    const authState = useAppSelector(selectAuthState)
+    const users = useAppSelector(selectUsersByID)
+    const isAuthenticated = (authState.currentUserID ? users[authState.currentUserID].is_superuser : false);
```

Comment:
Would rename isAuthenticated to hasWritePermission or something else to more specifically define the variable use.